docs/reference/module-overview.md

```diff
@@ -0,0 +1,284 @@
+
+Groups are added as an object with their name as the object key and the following properties:
+- `label`: The visible label (a string) on the interface tab
+- `fields`: An array of field names to include in the group
+
+The `@apostrophecms/doc-type` module groups the default fields in a `basics` group, which you can override. If overriding the `basics` group, you will need to add those core fields to a group in the module. Any fields not added to a group will be placed in an "Ungrouped" section in the editing interface.
+
+```javascript
+// modules/article/index.js
+modules.export = {
+  fields: {
+    add: {
+      // ...
+    },
+    group: {
+      meta: { // 👈 The group's identifying name is the object key.
+        label: 'Article metadata',
+        fields: [
+          'subtitle',
+          'author',
+          '_category'
+        ]
+      }
+    }
+  }
+};
+```
+
+### `filters`
+
+In piece type modules, the `filters` setting configures the pieces manager modal, adding and removing filtering interface (to view only certain pieces). `trash` and `visibility` filters are included by default.
+
+The `filters` object is configured with subsections: `add` and `remove`. Filters must correspond to an existing fields name or custom [query builder](#queries-self) on the piece type.
+
+#### `add`
+
+An object of filters to add to the piece type. Each filter is an object with its own configuration. Filter properties include:
+
+| Property | Description |
+| ------- | ------- |
+| `label` | Recommended, but not required, to customize the human-readable label. |
+| `inputType` | Choose an input field type for the filter from the following: `radio`, `checkbox`, or `select`. Defaults to `select`. |
+| `choices` | Manually set an array of choices instead of letting Apostrophe find the valid choices from the database. Choices include `label` and `value` properties. |
+| `def` | The default value for the manager filter. |
+
+```javascript
+// modules/article/index.js
+modules.export = {
+  filters: {
+    add: {
+      _category: { // 👈 Referencing a relationship field named `_category`
+        label: 'Article category'
+      },
+      featured: { // 👈 Referencing a boolean field name `featured`
+        labeled: 'Featured',
+        inputType: 'checkbox',
+        def: true,
+        choices: [
+          { value: true, label: 'Show featured' },
+          { value: false, label: 'Hide featured' }
+        ]
+      }
+    }
+  }
+};
+```
+
+#### `remove`
+
+An array of filter names from the base class module to remove.
+
+```javascript
+// modules/spotlight-article/index.js
+modules.export = {
+  extend: 'article',
+  filters: {
+    remove: [ 'featured' ]
+  }
+};
+```
+
+### `columns`
+
+In piece type modules, the `columns` setting configures the pieces manager modal, adding and removing the piece listing columns. Default columns include `title`, `updatedAt`, and `visibility`.
```

Comment:
"In piece type modules" -> "For piece-types"?

"the piece listing columns" While its true that these are displayed in columns, its not columns they are removing, its the data/info that is displayed (in columns). Fever thought: If we ever want to be mobile, these probably won't be displayed as `columns`. This option suffers from being too visually expressive in general.